sprinklers/base.py

```diff
@@ -29,6 +42,8 @@ class SprinklerBase(object):
     # Task limit per chord, no limit by default
     chord_size = None
     klass = None
+    shard_size = 100000
```

Comment:
Let's use 5000 or 10000 as default value. 100k is too big and we will have override it every time

Reply:
I tested it with up to 4x that and it worked fine on a 1x instance--it's not instantiating the models, since it's using the pk only. that being said, putting it to 10k won't hurt anything so I'll change it anyway.

Reply:
also I agree about chord_size--I just didn't want to clobber the sprinkler already using this. I figured when we merge this we can make sure the other one works. Since it changes the finished behavior I didn't want to assume it'd be OK to use the sharded version.